streamly-examples.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.2
 name:          streamly-examples
-version:       0.1.3
+version:       0.1.5
```

Comment:
We are not going to release it yet. So need not bump the version. We can just upload for docs update on the website.